RecoTracker/TrackProducer/plugins/TrackProducerFromPatMuons.cc

```diff
@@ -0,0 +1,51 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+
+class TrackProducerFromPatMuons : public edm::stream::EDProducer<> {
```

Comment:
why from `Pat`? it looks like regular `reco::Muons` can be used to be more generic; this would imply reading a `View<reco::Muon>`, but I think that the cost of making a view is acceptable